src/Sylius/Bundle/CoreBundle/Locale/TranslationLocaleProvider.php

```diff
@@ -37,7 +39,7 @@ public function __construct(LocaleContextInterface $localeContext)
      */
     public function getCurrentLocale()
     {
-        return $this->localeContext->getCurrentLocale();
+        return $this->container->get('sylius.context.locale')->getCurrentLocale();
```

Comment:
To avoid a problem with circular reference. I now, that it isn't the best solution, but only this work.